mountinfo/mountinfo_openbsd.go

```diff
@@ -0,0 +1,22 @@
+package mountinfo
+
+import "golang.org/x/sys/unix"
+
+func int8SliceToString(is []int8) string {
+	var bs []byte
+	for _, i := range is {
+		if i == 0 {
+			break
+		}
+		bs = append(bs, byte(i))
+	}
+	return string(bs)
```

Comment:
Sorry I overlooked it. This function is probably doing some excessive allocations and copying and should be rewritten

Reply:
Rewrited with `unsafe`.
https://github.com/moby/sys/pull/117